package/crds/ec2.aws.crossplane.io_subnets.yaml

```diff
@@ -28,6 +28,9 @@ spec:
             type: string
           availabilityZoneID:
             type: string
+          availableIPAddressCount:
```

Comment:
I think I saw this another time too, why is `availableIPAddressCount` getting into the spec here? Should it be a status-field?

Reply:
@chlunde in general the struct for subnet is like https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#Subnet and it is with availableIPAdressCount looks like the code generator introduced the status field since i added ec2 volume - possible that others also introduce the field with other ec2 services via code generator - is it a problem the field ? did i need to cover something in ec2 subnet about this field ?

![image](https://user-images.githubusercontent.com/21083497/134731257-d8f377fe-fb33-4e7e-a8ab-d89bf9d529bd.png)

Reply:
Here is the Problems 
Base: https://github.com/crossplane/provider-aws/issues/876

Reply:
fixed after #920